sdk/storage/Azure.Storage.Blobs/tests/BlobBaseClientTests.cs

```diff
@@ -693,7 +693,7 @@ public async Task DownloadAsync_VersionId()
             Assert.IsNotNull(response.Value.Details.VersionId);
         }
 
-        [PlaybackOnly("Object Replication policies is only enabled on certain storage accounts")]
+        [Ignore("Object Replication policies is only enabled on certain storage accounts")]
```

Comment:
what's the plan to bring back this test?

Reply:
We can't programmatically create accounts with ORS via the ARM templates for for the live tests.  We could re-record and make this [PlaybackOnly] if we can get an ORS account.  @amnguye